code-challenges.test.js

```diff
@@ -62,6 +128,29 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe("firstVowel", () => {
+    it("logs the index of the first vowel", () => {
+        expect(firstVowel(vowelTester1)).toEqual(1)
+        expect(firstVowel(vowelTester2)).toEqual(0)
+        expect(firstVowel(vowelTester3)).toEqual(2)
+    })
+})
+
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// name: firstVowel
+// input: string
+// output: number that logs the index of the first vowel
+// process: create a for loop that iterates through the string to detect if the string .includes a vowel, then return the index of the first instance, so include the return within the for loop
```

Comment:
This is not entirely accurate, as you aren't checking if the string includes a vowel, but rather if a string of vowels includes the current value of the iteration.